issue found on: content-service.go:145:2: importShadow

an issue has been detected in source code: ./content-service.go:145:2: importShadow: shadow of imported from 'github.com/RedHatInsights/insights-content-service/groups' package 'groups'

--- a/content-service.go
+++ b/content-service.go
@@ -142,14 +142,14 @@ func printVersionInfo() ExitCode {
 
 func printGroups() ExitCode {
 	groupsConfig := conf.GetGroupsConfiguration()
-	groups, err := groups.ParseGroupConfigFile(groupsConfig.ConfigPath)
+	groupsMap, err := groups.ParseGroupConfigFile(groupsConfig.ConfigPath)
 
 	if err != nil {
-		log.Error().Err(err).Msg("Groups init error")
+		log.Error().Err(err).Msg("Groups parsing error")
 		return ExitStatusServerError
 	}
 
-	fmt.Println(groups)
+	fmt.Println(groupsMap)
 	return ExitStatusOK
 }
 

--- a/server/server.go
+++ b/server/server.go
@@ -46,12 +46,12 @@ type HTTPServer struct {
 }
 
 // New constructs new implementation of Server interface
-func New(config Configuration, groups map[string]groups.Group,
+func New(config Configuration, groupsMap map[string]groups.Group,
 	contentDir content.RuleContentDirectory,
 	ruleContentStatusMap map[string]types.RuleContentStatus) *HTTPServer {
 	return &HTTPServer{
 		Config:               config,
-		Groups:               groups,
+		Groups:               groupsMap,
 		Content:              contentDir,
 		ruleContentStatusMap: ruleContentStatusMap,
 		InfoParams:           make(map[string]string),

